### ISSUE TITLE
Want to add Random Quote Website 

### ISSUE THREAD

**Issue by Purvesh77** [gauriiighadge/HactoberFest2023#139]
Hey @yysrll @Tushar-013 @Prayash77 @harshsingh32 @R0Y15 I want to add A JavaScript project for Hactoberfest !
Can you please assign me the task ?
Also add Hactoberfest-accepted tag on issue.
Thank You !

**Comment by R0Y15** [comment]
@Purvesh77 this repo has been excluded

**Comment by Purvesh77** [comment]
Ok @R0Y15 Can you suggest me another repository on which I can contribute for Hactoberfest !
Thank You !

**Comment by R0Y15** [comment]
@Purvesh77 you can search on github for "hacktoberfest" or "hacktoberfest2023" and select desired filters to get results in your favor, if you are interested in web do checkout my repos once (incase you wish to contribute in them) else there are plenty of repos available on github. 

**note: this is not a self promotion**

**Comment by Laxman67** [comment]
Yes ! I wanted to create Random Quote Generator Website that generate Random based on API

**Comment by kashyapanjali** [comment]
how to remove your repo on my github profile please give me
right solution
